README.md

```
@@ -1,253 +1,306 @@
+2. Click **+ Add Integration**
+3. Search for **"Garmin Connect"**
+4. Enter your configuration:
+   - **Username**: Your Garmin Connect username
+   - **Password**: Your Garmin Connect password
+   - **MFA Code**: Your Garmin Connect MFA code (optional)
+
+## Advanced Usage
+
+### Automation Examples
+
+**Add Body Composition**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)